fblldbobjcruntimehelpers.py

```diff
@@ -47,7 +47,7 @@ def functionPreambleExpressionForSelf():
   expressionForSelf = None
   if arch == 'i386':
     expressionForSelf = '*(id*)($esp+4)'
-  elif arch == 'x86_64':
+  elif arch == 'x86_64' or 'x86_64h':
```

Comment:
yeah, I can do that. I wasn't sure if there were any significant differences between x86_64 and x86_64h that would need to be addressed elsewhere in the code. This fix works for bmessage but I haven't tried out all the functionality in the repo so it may still break in other places.